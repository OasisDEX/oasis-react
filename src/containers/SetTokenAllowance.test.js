/* global shallow describe it expect */
/* eslint-disable import/first */
import React from 'react';
import Immutable from 'immutable';

import {
  SetTokenAllowanceWrapper,
  mapStateToProps,
  mapDispatchToProps
} from './SetTokenAllowance';
import { shallow } from 'enzyme';

describe('(Container) SetTokenAllowance', () => {
  const state = Immutable.fromJS({});
  const initialProps = mapStateToProps(state);
  const initialActions = mapDispatchToProps(x => x);
  const props = {
    ...initialActions,
    ...initialProps
  };

  it('will receive right props', () => {
    expect(initialProps).toMatchSnapshot();
  });


  it('will receive right actions', () => {
    expect(initialActions).toMatchSnapshot();
  });

  it('should render', () => {
    const wrapper = shallow(
      <SetTokenAllowanceWrapper {...props}/>
    );
    expect(wrapper).toMatchSnapshot();
  });

});
