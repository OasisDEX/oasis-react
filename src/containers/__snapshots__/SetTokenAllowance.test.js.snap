// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Container) SetTokenAllowance should render 1`] = `<div />`;

exports[`(Container) SetTokenAllowance will receive right actions 1`] = `
Object {
  "actions": Object {},
}
`;

exports[`(Container) SetTokenAllowance will receive right props 1`] = `Object {}`;
